libs/sysplugins/smarty_internal_errorhandler.php

```diff
@@ -66,7 +66,7 @@ public function deactivate() {
      */
     public function handleError($errno, $errstr, $errfile, $errline, $errcontext = [])
     {
-        if ($this->allowUndefinedVars && $errstr == 'Attempt to read property "value" on null') {
+        if ($this->allowUndefinedVars && preg_match('/^Attempt to read property ".+?" on null$/', $errstr)) {
```

Comment:
I want to use `\w` instead of `.`, when we need to catch alphanumeric case.

Reply:
Properties can also include `-` and maybe even `:` etc, so it **uneagerly** matches to the first `"`. https://3v4l.org/s9KPc

Reply:
It makes sense.